src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -137,7 +137,7 @@ nuget.config HTTP_PROXY</value>
     <comment>Please don't localize this string</comment>
   </data>
   <data name="DeleteCommandDescription" xml:space="preserve">
-    <value>Deletes a package from the server.</value>
```

Comment:
When you edit a resource file, please make sure the generated Designer file is added to the PR too.